org.scala-ide.sdt.core.tests/src/org/scalaide/core/testsetup/SDTTestUtils.scala

```diff
@@ -39,10 +39,11 @@ import org.eclipse.jdt.launching.JavaRuntime
 import org.scalaide.core.compiler.IScalaPresentationCompiler
 import org.scalaide.core.internal.project.ScalaProject
 
-/** Utility functions for setting up test projects.
+/**
+ * Utility functions for setting up test projects.
  *
  */
-object SDTTestUtils extends HasLogger {
+trait SDTTestUtils extends HasLogger {
```

Comment:
it looks like a kind of util for integration test creation. when I look at scalatest or specs2 such utilities are traits usually not objects to import... I change it if it sores your eyes